Competition2018/src/main/java/competition/subsystems/climberdeploy/ClimberdeploySubsystem.java

```diff
@@ -1,33 +1,50 @@
 package competition.subsystems.climberdeploy;
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 
 import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 
 @Singleton
 public class ClimberdeploySubsystem extends BaseSubsystem {
 	
 	DoubleProperty deploySpeed;
+	CommonLibFactory clf;
+	double speed;
+	
+	public XCANTalon motor;
+
 	
 	@Inject
-	public ClimberdeploySubsystem(XPropertyManager propMan) {
-		propMan.createPersistentProperty("deploySpeed", 0.2);
+	public ClimberdeploySubsystem(CommonLibFactory clf, XPropertyManager propMan) {
+		this.clf = clf;
+		deploySpeed =  propMan.createPersistentProperty("deploySpeed", 0.2);
```

Comment:
should be named something like ClimberDeploySpeed, since "deploy" is a generic verb that might apply to other subsystems